public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -160,14 +168,16 @@ class UnThemedLogRowMessage extends PureComponent<Props> {
           <span className={cx(styles.positionRelative, { [styles.rowWithContext]: contextIsOpen })}>
             {renderLogMessage(hasAnsi, restructuredEntry, row.searchWords, style.logsRowMatchHighLight)}
           </span>
-          {!contextIsOpen && showContextToggle?.(row) && (
+          {showRowMenu && (
             <span
-              className={cx('log-row-context', style.context, styles.contextButton)}
+              className={cx('log-row-menu', styles.rowMenu, showContextToggle?.(row) ? styles.large : styles.small)}
               onClick={(e) => e.stopPropagation()}
             >
-              <Tooltip placement="top" content={'Show context'}>
-                <IconButton size="md" name="gf-show-context" onClick={this.onContextToggle} />
-              </Tooltip>
+              {showContextToggle?.(row) && (
```

Comment:
Not related to your PR, but maybe we could extract `showContextToggle?.(row)` into variable, cause this is quite hard to read/figure out what it means?

Reply:
Good idea, changed in 38c8334e25

Reply:
Yea good idea. Done in 38c8334e2593404754f77dc6e28509e461315109